content/20.treatments.md

```diff
@@ -105,6 +105,49 @@ Summarize the state of the antiviral approach.
 #### Broad-Spectrum Pharmaceuticals
 
 ##### Nutraceuticals
+Considering the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies for which nutraceuticals hold some promise.
+Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value designed for the prophylaxis and treatment of disease [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
+Nutraceuticals purported to boost the immune response, reduce  immunopathology, exhibit antiviral activities or prevent acute respiratory distress syndrome (ARDS) are being considered for their potential therapeutic value [@doi:ggkd3b].
+A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
+These include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
+Considerable evidence in vitro and in vivo suggests that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of toll-like receptor 7 (TLR7) [@doi:10.1016/j.pcad.2020.02.007].
+
+Another potential nutraceutical that has exhibited beneficial effects against various viral infections is n-3 PUFA [@doi:10.1002/jmv.25707], such as eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
+N-3 PUFA nutraceuticals can mediate inflammation and they have the capacity to modulate the adaptive immune response [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.3390/molecules22111964; @doi:10.1038/nrcardio.2016.103].
```

Comment:
You are definitely the expert here, especially as you authored one of the citations, but the first reference made it sound like the research showing this has largely been in animal models? I just wanted to check if that's true.

Reply:
Thanks for checking. The majority of the mechanistic research has largely been in animal models. There is mixed research in this field because the potential beneficial effects of n-3 fatty acids is very pathogen specific, in some cases it is suggested they do nothing, in others they might do more harm than good. In terms of coronavirus there is promise because of the cardiovascular pathology involved. But on that point I might see if I can find a few human studies or a meta-analysis for here